packages/docs/site/docusaurus.config.js

```diff
@@ -198,7 +200,7 @@ const config = {
 			},
 		}),
 };
-
+console.log(JSON.stringify(config, null, 2));
```

Comment:
@juanmaguitar I take it this is intentional to log the entire config?

Reply:
Nope, that shouldn't be there. I just comitted the removal of that line.
Thanks for the heads up!